server/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "tsx src/index.ts"
```

Small suggestion: enable live reload

```suggestion
    "dev": "tsx watch src/index.ts"
```

---

good point